paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -803,6 +810,51 @@ internal abstract class BaseSheetViewModel(
         eventReporter.onCardNumberCompleted()
     }
 
+    fun usBankAccountFormArguments(selectedPaymentMethodCode: String): USBankAccountFormArguments {
```

Comment:
is there a better place to put this than in BaseSheetViewModel (to avoid making this even larger)? E.g. could this be a UsBankAccountFormArguments constructor that takes a selectedPaymentMethodCode as its only param?

Reply:
I could probably move this to `UsBankAccountFormArguments`, but it would need basesheetviewmodel too. I'll update.